### ISSUE TITLE
Forward all arguments with `Style/ArgumentsForwarding`

### ISSUE THREAD

**Issue by teoljungberg** [rubocop/rubocop#10090]
We have uses of `*args` and `**args` being forwarded in our codebase. They occur like

```ruby
def self.call(*args)
  new(*args).call
end
```

or 

```ruby
def self.call(**args)
  new(**args).call
end
```

I ideally want them to both be collapsed into

```ruby
def self.call(...)
  new(...).call
end
```

I wanted to enable `Style/ArgumentsForwarding` to prefer using `...` instead, however, I found that enabling `Style/ArgumentsForwarding` did not catch replacing `*args` with `...`. I then tried it with the configuration option `AllowOnlyRestArgument: false` on the cop and then `*args` are suggested to be replaced with `...`.

I could not find an option that suggests replacing `**args` with `...`.

**Comment by teoljungberg** [comment]
Thank you @koic

**Comment by koic** [comment]
Thank you for the feedback @teoljungberg!
